### DISCUSSION TITLE
Phantom Registration Error >5mm

### DISCUSSION THREAD

**Discussion by LeidyMoraV** [PlusToolkit/PlusLib#1190] [Q&A]
Hello, I'm currently working on calibrating the Ascension Trackstar + Epiphan (ultrasound) system. I've noticed that my spatial calibration error typically hovers around 6mm. I suspect that this error might be linked to the phantom registration process itself. While the stylus error stands at 0.20mm, the landmarks error is considerably higher at 6.94mm. Could you suggest methods or techniques to reduce this calibration error?

I'm using the Perklab Phantom fcal 2.0 with 3Wires

![image](https://github.com/PlusToolkit/PlusLib/assets/162748695/b2c5b7ee-5d13-49e6-ad88-7157dac07742)

config file: https://pennstateoffice365-my.sharepoint.com/:u:/g/personal/lfm5705_psu_edu/ESbY2FjRy8VIrKbnpP3AESkB-P-vPsTvQnktV5xJ3dIkMg?e=prQsRx

**Comment by kenavolic** [comment]
When we did calibration with aruco tracker, we needed a lot of attempts to get acceptable results. At the end, we decreased the phantom to cam distance and got better results.
